app/package.json

```diff
@@ -12,7 +12,9 @@
   "license": "Apache-2.0",
   "dependencies": {
     "datomish-user-agent-service": "0.0.2",
+    "fs-promise": "0.5.0",
     "request": "2.65.0",
+    "thenify": "3.2.0",
```

Comment:
I wish we could add comments to the json to explain them. It's basically for packages that webpack fails to package for some reason. In this case thenify relies on any-promise which attempts to require a whole bunch of promise modules that we don't have installed which was causing webpack to fail.